### DISCUSSION TITLE
May 21 Writers workshop

### DISCUSSION THREAD

**Discussion by tonyfast** [Quansight/writers-workshop#8] [New class]
# the goal of the workshop

the goal of the workshop is to talk about critique of literate programs

## teacher's preferred ground rules

* How to use the raise handle feature? Keeping the professor on track..
* I encourage y'all to interrupt me when I'm talking or demo'ing. 
  Sometimes I get distracted by my own special interests and this class if for you all.
* To help take notes, please respond to this discussion item and continue editing your one reply during class. i can see the changes on the discussion board as they happen.

## [call to action] :+1: :-1:  :smile: :confused:  :rocket:  :heart:  :eyes: :tada: attendance

react to this discussion so we can take attendance.

![image](https://user-images.githubusercontent.com/4236275/117461813-fd163180-af1b-11eb-901b-bfdee65d8bdc.png)

### Q1 how's your energy

* :+1: i feel energetic from this week
* :-1: i feel exhausted from this week

### Q2 how do you want to participate

* :heart:   i'm happy to be at the writers workshop this week
* :eyes: i want to just watch and listen
* :tada: i am okay being called on for discussions

## agenda

0. [:00-:07] chit chat
1. [:07-:12] Recap and introduce new folks
    
    [Preview of our class docs!](https://quansight-writers-workshop.readthedocs.io/en/latest/qww/short%20programs.html)

2. [:12-:40] discuss the homework _and_ [submit an idea you'd like to read about](https://github.com/Quansight/writers-workshop/issues/new?assignees=&labels=idea%2C&template=suggestion.md&title=%F0%9F%92%A1+i+want+to+read+about+...).
3. [:40-:50] review some ideas submitted

  ## assignment

4. [:50-:60] wrapup

    edit and improve your story in the third person, separate yourself from your work.

**Comment by MarsBarLee** [comment]
Writing Styleguides
- [Mailchimp](https://styleguide.mailchimp.com/)
- Writing for everyone:
-- Words List for [Standardized Spellings ](https://styleguide.mailchimp.com/word-list/)
-- Avoid Words that are jargon and ableist

Tony's Writing Inspirations
- The Pleasure of the Text  [PDF](https://emberilmu.files.wordpress.com/2011/08/roland-barthes-the-pleasure-of-the-text.pdf)
- Perlis Alan's [EPIGRAMS IN PROGRAMMING](http://www.cs.yale.edu/homes/perlis-alan/quotes.html)
